keep-only-warm-data/README.md

```diff
@@ -0,0 +1,125 @@
+# Keep only warm data into Kuzzle
+
+## Requirements
+
+Kuzzle : `>= 1.2.11`  
+Elasticsearch : `>= 5.4.1`  
+Redis : `>= 3.2`  
+
+## Introduction
+
+Kuzzle is capable of managing a very large number of documents and offering very good performances.  
+However in a Big Data context, it can be interesting to rationalize the cost of the infrastructure necessary to Kuzzle by keeping on Kuzzle only the relevant data.  
+Coupled with a secondary database, Kuzzle can be used in a Hot/Warm architecture where the most recent data is kept on Kuzzle and the oldest data in a secondary DBMS.  
+
+In most cases, deleting your oldest data from Kuzzle will only make sense if you have previously dumped it into another database. (see [How-To Synchronize Kuzzle with another database](../sync-data-to-another-database))
+
+## Architecture
+
+We will be using the standard Kuzzle stack (Kuzzle, Elasticsearch and Redis). (Check [docker-compose.yml](docker-compose.yml) for more details)
```

Comment:
-"standard Kuzzle stack" +"open-source Kuzzle stack"

Reply:
I would also remove the `(Kuzzle, Elasticsearch and Redis)` part, but that might be just me

Reply:
Yes, they can check docker-compose.yml for more details.